tests/TestCase/Http/Middleware/BodyParserMiddlewareTest.php

```diff
@@ -361,6 +376,8 @@ public function testInvokeXmlInternalEntities()
         $response = new Response();
         $next = function ($req, $res) {
```

Comment:
We should also start the habit of typehinting closures, this way everyone and tooling can look into the object methods used there. Especially in tests this doesnt hurt. But can also be another follow up in the future.